watcher/lib/influxdb/rollups.go

```diff
@@ -176,7 +176,7 @@ func (f *Function) buildFunction() (string, error) {
 	return fmt.Sprintf(`%v("%v", %v) as %v`, funcName, f.Field, param, alias), nil
 }
 
-// Check verifies the function configuration is correct
+// isComposite checks if the function is composite
```

Comment:
`isComposite returns true if this function is one of the functions that require additional parameters`